pages/dashboard/views/index.jsx

```diff
@@ -4,20 +4,14 @@ import {
   Link,
   Snippet,
   Header,
-  PanelList
+  PanelList,
+  ExpandingPanel
 } from '@asl/components';
 import TaskList from '@asl/pages/pages/task/list/views/tasklist';
-
-const EstablishmentPanel = ({ id, name }) => (
-  <Fragment>
-    <p>
-      <Link page="establishment.dashboard" establishmentId={id} label={name} />
-    </p>
-    <p>
-      <Link page="profile.invite" establishmentId={id} label={<Snippet>pages.dashboard.invite</Snippet>}/>
-    </p>
-  </Fragment>
-);
+import format from 'date-fns/format';
+import { isEmpty, sortBy } from 'lodash';
```

Comment:
```
import isEmpty from 'lodash/isEmpty';
import sortBy from 'lodash/sortBy';
```
we don't want the whole of lodash in out client bundle

Reply:
done